.github/workflows/create_release.yml

```diff
@@ -27,6 +24,13 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Install transcrypt
+        run: |
+          mkdir -p $HOME/.local/bin
+          echo "$HOME/.local/bin" >> $GITHUB_PATH
+          wget https://raw.githubusercontent.com/elasticdog/transcrypt/refs/heads/main/transcrypt -O "$HOME/.local/bin/transcrypt"
```

Comment:
How about linking to a stable tag? What if transcrypt the `main`branch evolves and breaks its API?